packages/fiori/src/SideNavigationSelectableItemBase.ts

```diff
@@ -0,0 +1,173 @@
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import { isSpace, isEnter } from "@ui5/webcomponents-base/dist/Keys.js";
+import SideNavigationItemBase from "./SideNavigationItemBase.js";
+
+/**
+ * Fired when the component is activated either with a
+ * click/tap or by using the Enter or Space key.
```

Comment:
click/tap or by using the [Enter] or [Space] keys.

Reply:
Done